index.html

```diff
@@ -412,7 +412,7 @@
             The list of valid post properties is defined by the vocabularies. This allows new vocabularies to be developed outside the development of the syntax.
           </p>
           <p>
-            Most values will be strings. If a property (such as `author` for example) references another
+            Most values will be strings. If a property (such as <a>author</a> for example) references another
```

Comment:
hmm, good point, i'm not sure, it might be better to just make those a different case, as that would create a lot of outbound links to things that might move around.  So long as its clear where to go to look up the info about it in the context, i don't see them needing to be links